mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -880,6 +900,12 @@ func resourceStorageBucketUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if d.HasChange("hierarchical_namespace") {
+		if v, ok := d.GetOk("hierarchical_namespace"); ok {
+			sb.HierarchicalNamespace = expandBucketHierachicalNamespace(v.([]interface{}))
```

Comment:
I believe, There is no need for checking this block in update as the block is marked with ForceNew which will never allow having updates for the block.

Reply:
removed this block